src/brouwers/kitreviews/templates/kitreviews/review_form.html

```diff
@@ -1,57 +1,25 @@
-{% load i18n %}
+{% load i18n sniplates %}
 
-<div id="review_kit">
-    {% with reviewform.model_kit as field %}
-        {% include "kitreviews/review_form_field.html" %}
-    {% endwith %}
-</div>
+{% block form %}
 
-<div id="review_raw_text">
-    {% with reviewform.raw_text as field %}
-        {% include "kitreviews/review_form_field.html" %}
-    {% endwith %}
-</div>
+    <fieldset class="well">
+        {% load_widgets form='horizontal.form' %}
+        {% form_field form.model_kit %}
```

Comment:
later on this will change so that it's pre-populated I think, have to come up with a nice way to do that. I think we can reuse parts of the 'builds' app there with the form field for `kits` on the `Build` model.